openapiart/tests/field_uid/property_name.yaml

```diff
@@ -0,0 +1,64 @@
+paths:
+  /config:
+    post:
+      tags: ['Bundler']
+      operationId: set_config
+      description: >-
+        Sets configuration resources.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Field.Config'
+      responses:
+        '200':
+          x-field-uid: 1
+          description: 'Standard success binary payload'
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                format: binary
+        '400':
+          x-field-uid: 2
+          description: 'error 4xx'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  errors:
+                    type: array
+                    items:
+                      type: string
+                    x-field-uid: 1
+        '500':
+          x-field-uid: 4
+          description: 'error 5xx'
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  errors:
+                    type: array
+                    items:
+                      type: string
+                    x-field-uid: 1
+components:
+  schemas:
+    Field.Config:
+      type: object
+      description: |-
+        This is a field uid object
+      properties:
+        camelCase:
+          type: boolean
+          x-field-uid: 1
+        PascalCase:
+          type: boolean
+          x-field-uid: 2
+        Upper_case:
+          type: boolean
+          x-field-uid: 3
```

Comment:
Is the test actually exercising all three use cases ?

Reply:
Only the 1st one will be tested but the remaining are for future users to understand what we are not supporting.

Reply:
done